x-pack/plugins/synthetics/server/legacy_uptime/lib/requests/get_monitor.ts

```diff
@@ -23,7 +23,7 @@ export const getSyntheticsMonitor = async ({
   monitorId: string;
   encryptedSavedObjectsClient: EncryptedSavedObjectsClient;
   savedObjectsClient: SavedObjectsClientContract;
-}): Promise<SavedObject<SyntheticsMonitor>> => {
+}): Promise<SyntheticsMonitor> => {
```

Comment:
Unrelated to your PR, but I[m unsure of why this file is in `legacy_uptime`. @shahzad31